manifests/09-prometheus-rules.yaml

```diff
@@ -28,3 +28,26 @@ spec:
           DeploymentConfigs which reference ImageStreamTags may not work as
           expected. Please configure storage and update the config to Managed
           state by editing configs.imageregistry.operator.openshift.io.
+  - name: ImagePruner
+    rules:
+    - alert: ImagePruningDisabled
+      expr: image_registry_operator_image_pruner_install_status < 2
+      labels:
+        severity: warning
+      annotations:
+        message: |
+          Automatic image pruning is not enabled. Regular pruning of images
+          no longer referenced by ImageStreams is strongly recommended to
+          ensure your cluster remains healthy.
+          
+          To remove this warning, install the image pruner by creating an
+          imagepruner.imageregistry.operator.openshift.io resource with the
+          name `cluster`. Ensure that the `suspend` field is set to `false`.
+    - alert: ImagePruningFailed
+      expr: increase(image_registry_operator_image_pruner_completed_jobs_total{result="failed"}[24h]) > 0
```

Comment:
fyi @openshift/openshift-team-monitoring - this is a new alert driven by a new metric. The metric is a counter with one label which should only have two values (2 new time series).